components/kyma-environment-broker/internal/process/upgrade_kyma/connectivity_bind_test.go

```diff
@@ -1,101 +1,122 @@
 package upgrade_kyma
 
 import (
-	"encoding/json"
 	"testing"
 
-	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/process/provisioning"
-
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/logger"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/process/connectivity_bind"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/process/provisioning/automock"
 	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/servicemanager"
-
-	"github.com/stretchr/testify/assert"
+	"github.com/kyma-project/control-plane/components/provisioner/pkg/gqlschema"
+	"github.com/stretchr/testify/require"
 )
 
-const connectivityServiceKey = `
-	{
-		"clientid" : "clientid314159265359",
-		"clientsecret": "Y2xpZW50c2VjcmV0Cg==",
-		"connectivity_service":
+func TestUpgradeConnectivityBind_Run(t *testing.T) {
```

Comment:
Please see my comments to `provisioning/connectivity_bind_test.go` I think we should test here more logic paths. First pass/second pass/ retry.